tests/OTAnalytics/application/use_cases/test_event_repository.py

```diff
@@ -48,3 +54,29 @@ def test_remove_events_of_changed_sections(self) -> None:
         )
 
         repository.remove.assert_called_once_with([section_1])
+
+
+class TestRemoveEventsByRoadUserId:
+    def test_remove_multiple(self) -> None:
+        given_event_repository = Mock(spec=EventRepository)
+        given_road_user_ids = [TrackId("1"), TrackId("2")]
+        target = RemoveEventsByRoadUserId(given_event_repository)
+        target.remove_multiple(given_road_user_ids)
+
+        given_event_repository.remove_events_by_road_user_ids.assert_called_once_with(
+            given_road_user_ids
+        )
+
+    @patch(
+        (
+            "OTAnalytics.application.use_cases.event_repository."
```

Comment:
Why do we need to patch the method here?

Reply:
I assumed the underlying call to `remove_multiple` to work correctly to avoid duplication, since it has also been tested. See `TestRemoveEventsByRoadUserId.test_remove_multiple`.